docs/development.md

```diff
@@ -185,41 +188,49 @@ Before running the test suite, both the server and the test environment should b
+### Running the test suite with command line
 
-### Running the test suite in an IDE
+Before running any test suite, it is important to have the dev server running locally first if you are testing against it.
 
-When running the test cases, if a few cases fail (this can happen due to timing issues), rerun the failed cases using the `Run Failed Test` icon in the TestNG tab until they pass.
+Test suite | Command | Results can be viewed in
+---|---|---
+`CI tests` | `./gradlew ciTests` | `{project folder}/build/reports/test-try-{n}/index.html`, where `{n}` is the sequence number of the test run
+`Local tests` | `./gradlew localTests` | `{project folder}/build/reports/test-local/index.html`
+`Failed tests` | `./gradlew failedTests` | `{project folder}/build/reports/test-failed/index.html`
+Any individual test | `./gradlew test -Dtest.single=TestClassName` | `{project folder}/build/reports/tests/index.html`
 
-#### Eclipse
+`CI tests` will be run once and the failed tests will be re-run a few times.
+All other test suites will be run once and only once.
 
-Run tests using the configurations available under the green `Run` button on the Eclipse toolbar.
+To run any test suite or individual test with [GodMode turned on](godmode.md), append `-Pgodmode=true` to the command, e.g.:
+```sh
+./gradlew ciTests -Pgodmode=true
+./gradlew test -Dtest.single=InstructorFeedbackResultsPageUiTest -Pgodmode=true
+```
 
-Sometimes, Eclipse does not show these options immediately after you set up the project. "Refreshing" the project should fix that.
+### Running the test suite with an IDE
 
-#### IntelliJ
+* An additional configuration `All tests` is provided, which will run `CI tests` and `Local tests`.
+* Additionally, configurations that run the tests with `GodMode` turned on are also provided.
+* When running the test cases, if a few cases fail (this can happen due to timing issues), re-run the failed cases using the `Run Failed Test` icon in the TestNG tab until they pass.
 
-Run tests using the configurations available under `Run → Play`.
+#### Eclipse
+
+Run tests using the configurations available under the green `Run` button on the Eclipse toolbar.
 
-### Running the test suite outside Eclipse and IntelliJ
+Sometimes, Eclipse does not show these options immediately after you set up the project. "Refreshing" the project should fix that.
 
-Typically, we run the test suite within an IDE, but core developers may prefer to run it outside the IDE so that they can continue to use their IDE while the test suite is running. If you wish to do such, given below is the procedure:
+To run individual tests, right-click on the test files on the project explorer and choose `Run As → TestNG Test`.
 
-1. Start the dev server.
+#### IntelliJ
 
-1. Run the following command to run the full test suite once and retry the failed tests several times:
-   ```sh
-   ./gradlew ciTests
-   ```
+Run tests using the configurations available under `Play → Run`.
```

Comment:
Oops looks like my correction was not totally clear, it should be exactly `Run → Run...` with the ellipses.